docs/api-gateway-v2/03-04-migration.md

```diff
@@ -1,38 +1,38 @@
 
->**CAUTION:** Some Api CR specifications are too complex or do not meet all requirements to pass the automatic migration. To ensure all your services are exposed using APIRule CRs, [verify the migration outcome](#verify-automatic-migration). If there still are Api CRs in use, follow [this](#manual-migration) guide to migrate them.
+During migration, it may turn out that some resource specifications are too complex or fail to meet all the migration requirements. In such a case the process skips them but doesn't break the way existing services are exposed. However, if you want to introduce further changes or remove the Api CR, your actions won't have any effect on how the service is exposed because it will still use the original configuration.  
 
+>**NOTE:** If the migration process skipped the Api resources due to the invalid status or a blacklisted label, migration is not possible.
 
-Skipping some resources during migration will not break the way existing services are exposed. However, bear in mind that if you want to introduce changes or remove the Api CR, it won't have any effect on how the service is exposed because the service will still use the original configuration. 
+Here are the steps you need to follow to ensure your services are properly migrated:
+1. [Verify the migration outcome](#verify-automatic-migration). 
+2. If you can still see any Api CRs in use, use the [manual migration](#manual-migration) guide to migrate them.
 
+## Verify the automatic migration
 
-## Verify automatic migration
-
-Follow these steps to verify if all the Api CRs were migrated to APIRule CRs:
+Follow these steps to verify if all Api CRs were migrated to APIRule CRs.
 
 1. Once the Kyma upgrade finishes, list the existing Api CRs:
 
     ```shell script
       kubectl get apis --all-namespaces
     ```
 
-2. As Api CRs are removed after a successful migration, the list shows all resources that the migration job skipped. Fetch the logs from the job's Pod to learn the reasons for this behavior: 
+2. As Api CRs are removed after a successful migration, the list shows all resources skipped by the migration job. Fetch the logs from the job's Pod to find out the reasons: 
 
     ```shell script
     kubectl logs api-gateway-api-migrator-job -n kyma-system
     ```
 
->**NOTE:** If the migration process skipped the Api resources due to the invalid status or a blacklisted label, you either should not or will not be able to migrate them.
-
 ## Manual migration
 
-This guide shows how you can manually migrate Api resources to API Rules.
+This guide shows how you can manually migrate Api CRs to APIRule CRs.
 
 >**NOTE:** Before you start the manual migration process, see the [Api CR](/components/api-gateway/#custom-resource-api-sample-custom-resource) and [APIRule CR](/components/api-gateway-v2#custom-resource-api-rule) documents for the custom resource detailed definition.
```

Comment:
The problem with the link `[Api CR](/components/api-gateway/#custom-resource-api-sample-custom-resource)` is that it will not work after merging this PR, as we are removing that part of the documentation. Previously I used links to specific version of the documentation (https://kyma-project.io/docs/1.11/components/api-gateway#custom-resource-api-sample-custom-resource) but I see that you changed it. It applies to all links referring to the api-controller (api-gateway v1) documentation.